test/DebuggerCommon/ES6_proto_invalidation.js.dbg.baseline

```diff
@@ -461,10 +461,9 @@
             },
             "indexOf": {
               "#__proto__": "function <large string>",
-              "length": "number 1",
+              "prototype": "Object {...}",

```

Comment:
This change in behavior is a bit concerning. Can you double check the value of the prototype before and after your change?